### PR TITLE
New Files Added : Asignment1, assignment2 and  assignment3

### PR DESCRIPTION
**Pull request by vinod2406**

### CODE REVIEW

**Review by vishnukopnar — APPROVED** [review]

**Review by Pranjali1999 — APPROVED** [review]

**Review by akash19972306 — APPROVED** [review]

**Review by vishnukopnar — APPROVED** [review]

File: vinod/html/ASSIN2/assingment2.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+      <h2>Initial public offering</h2>
+
+      Google intial public offering(IPO)took place five year later,on 19th August 2004.<br>
+      At that time Larry page,Sergey Brin,and Eric Schmidt agreed to work together at Google for 30 <u>20</u> 
+      year, unit the year 2024.<br>
+      
+     <center> <font color="green" ><h2> Thank You!!</h2></font></center>
+      
+
```
Comment (shivling123) [inline vinod/html/ASSIN2/assingment2.html]: format code properly remove extraspace

File: vinod/html/ASSIN3/assingment3.q2.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    </style>
+</head>
+
+<body>
+
+    <center>
+        <h1>Ambersoft</h1>
+    </center>
+    </div>
+    <center> <a href="assingment3.q2.html"> Home </a> | <a href="link.html"> MI-MEAN </a> | <a href="link1.html">
```
Comment (shivling123) [inline vinod/html/ASSIN3/assingment3.q2.html]: format it properly

File: vinod/html/ASSIN3/assingment3.q2.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            aliquam, dolore ipsum esse consequuntur quod facere omnis fugit? Odit, veritatis nobis ea ex aut, beatae
+            explicabo similique praesentium laborum voluptatibus quo sint vel quos sed nihil sapiente incidunt natus
+            repellat maxime ratione fugit. Libero ea odio ad non, corrupti ut cupiditate tempore quibusdam voluptatem
+            nulla dolor sit tempora, explicabo laboriosam iste quia officiis porro saepe cum! Distinctio vero ipsum
+            quam, ut a aperiam esse debitis molestiae nobis dolorem perferendis, porro cumque? Nobis dolore nulla rerum.
+            Maxime consequatur quisquam iste, ducimus ad aperiam culpa quae voluptatibus iusto possimus id dolor? Dolor
+            officia quis sed illo quia esse incidunt, fugit obcaecati facere repellat odit ut nisi rerum voluptatum,
+            accusantium hic itaque ducimus quaerat provident neque tempora dolorum!</p>
+       
+    </div></div></div>
```
Comment (shivling123) [inline vinod/html/ASSIN3/assingment3.q2.html]: indentation not proper
close div tag on new line

File: vinod/html/ASSIN4/assingment4.q2.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    <td></td>
+    <td></td>
+</tr>
+<tr>
+    <td>3:30</td>
+    <td rowspan="4">CSC 205</td>
+    <td rowspan="3" >CSC</td>
+    <td rowspan="3">csc 205</td>
+    
+
```
Comment (shivling123) [inline vinod/html/ASSIN4/assingment4.q2.html]: remove extra space format code properly

File: vinod/html/ASSIN4/assingment4.q2.html [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+    <td rowspan="3">EC metting</td>
+    <td></td>
+    <td></td>
+</tr>
+<tr>
+    <td>3:30</td>
+    <td rowspan="4">CSC 205</td>
+    <td rowspan="3" >CSC</td>
+    <td rowspan="3">csc 205</td>
+
```
Comment (shivling123) [inline vinod/html/ASSIN4/assingment4.q2.html]: remove extra space

File: vinod/html/assignment1/assingment1.q2.html [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    <h1>HTML</h1>
+</body>
+</html>
+
+<h3>INTRODUCTION</h3>
+HTML is standaed markup language for web page.
+<h1>HTML Editor</h1>
+<h3>Notpad or TextEdit</h3>
+web page can be created and modified by using professional HTML editors.<br>
+However,for learing HTML we recommend a simple text editor like Notpad(pc) or TextEdit(Mac).
```
Comment (shivling123) [inline vinod/html/assignment1/assingment1.q2.html]: add closing tag body ..etc

File: vinod/html/assignment1/assingment1.q3.html [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>HTML</h1>
+    <h3>INTRODUCTION</h3>
+HTML is standaed markup language for web page.
```
Comment (shivling123) [inline vinod/html/assignment1/assingment1.q3.html]: containt should be inside tag

**Review by shivling123 — CHANGES_REQUESTED** [review]

**Review by shivling123 — CHANGES_REQUESTED** [review]
make changes in code as per required

File: vinod/html/ASSIN3/assingment3.q2.html [hunk 8]
```diff
@@ -12,10 +11,8 @@
         }
```
Comment (shivling123) [inline vinod/html/ASSIN3/assingment3.q2.html:11]: check opening of style tag

**Review by shivling123 — APPROVED** [review]

**Review by shivling123 — APPROVED** [review]

**Review by shivling123 — APPROVED** [review]

**Review by shivling123 — APPROVED** [review]

File: vinod/html/ASSIN3/assingment3.q2.html [hunk 9]
```diff
@@ -12,10 +11,8 @@
...
     </style>
 </head>
-
 <body>
-
-    <center>
+   <center>
         <h1>Ambersoft</h1>
     </center>
     </div>
```
Comment (shivling123) [inline vinod/html/ASSIN3/assingment3.q2.html]: where is opening div tag
check it and update

**Review by shivling123 — APPROVED** [review]

**Review by shivling123 — APPROVED** [review]

**Review by shivling123 — APPROVED** [review]

File: vinod/html/ASSIN2/assingment2.html [hunk 10]
```diff
@@ -1,77 +1,65 @@
...
-      <h2>Initial public offering</h2>
-
-      Google intial public offering(IPO)took place five year later,on 19 th August 2004.<br>
-      At that time Larry page,Sergey Brin,and Eric Schmidt agreed to work together at Google for <del>30</del> <u>20</u> <br>
-      year, unit the year 2024.<br>
+<body>
+  <center>
+    <h1>google</h1>
+  </center>
+  This artical is about the company.<br>
```
Comment (shivling123) [inline vinod/html/ASSIN2/assingment2.html]: add this containt  inside proper tag tag

File: vinod/html/ASSIN2/assingment2.html [hunk 11]
```diff
@@ -1,77 +1,65 @@
...
+    <b>Larry Page</b><i>was consadering Standford for gard school and </i><b>Sergey Brin,</b><i> a student</i>
+    <i>there,was assigned to show him around</i><br>
+    By some account they disagreed about nearly everything that first meeting,but by following
+    year they struck a <mark>partnership</mark>.<br>
+    Working form their dorm room,they built a search engine that used links to determine the importance of
+    individual pages on the <font color="blue">World Wide Web.</font> <br>
+    They called this search engine <font color="black" style=background-color:blue>Backrub</font> <br>
+    <hr font color="blue">
+    <h2>Initial public offering</h2>
+    Google intial public offering(IPO)took place five year later,on 19th August 2004.<br>
```
Comment (shivling123) [inline vinod/html/ASSIN2/assingment2.html]: add this containt inside tag

**Comment by avinash0070** [comment]
need proper title ask with sourabh kanwade 
in title mention it which file or folder you added
and
assign yourself
do this changes

File: vinod/html/ASSIN2/assingment2.html [hunk 12]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (avinash0070) [inline vinod/html/ASSIN2/assingment2.html:3]: remove all extra lines 
need proper code indentation 
use 4 space in code 
use prettier setting in vs code ask with your mentor

File: vinod/html/ASSIN2/assingment2.html [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+      At that time Larry page,Sergey Brin,and Eric Schmidt agreed to work together at Google for 30 <u>20</u>
+      year, unit the year 2024</P>.<br>
+      </div>
+     <center>
+      <font color="green">
+        <h2> Thank You!!</h2>
+     </font>
+    </center>
+  </body>
+</html>
```
Comment (avinash0070) [inline vinod/html/ASSIN2/assingment2.html]: add new line

File: vinod/html/ASSIN3/assingment 3.q1.html [hunk 14]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline vinod/html/ASSIN3/assingment 3.q1.html:1]: remove all extra lines
need proper code indentation
use 4 space in code

File: vinod/html/ASSIN3/assingment 3.q1.html [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+                <ul type="disc">
+                    <li>triveni
+                   <li>kalyani
+                    </ul>   
+    </ol>
+    
+    
+
+</body>
+</html>
```
Comment (avinash0070) [inline vinod/html/ASSIN3/assingment 3.q1.html]: add extra line

File: vinod/html/ASSIN3/assingment3.q2.html [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+            quam, ut a aperiam esse debitis molestiae nobis dolorem perferendis, porro cumque? Nobis dolore nulla rerum.
+            Maxime consequatur quisquam iste, ducimus ad aperiam culpa quae voluptatibus iusto possimus id dolor? Dolor
+            officia quis sed illo quia esse incidunt, fugit obcaecati facere repellat odit ut nisi rerum voluptatum,
+            accusantium hic itaque ducimus quaerat provident neque tempora dolorum!</p>
+        </div>
+        <h2>Home Page Subheading</h2>
+        this is home page subheading.this is home page subheading.this is home page subheading. this
+        is home page subheading. this is home page subheading.this is home page subheading
+        </body>
+    </html>
```
Comment (avinash0070) [inline vinod/html/ASSIN3/assingment3.q2.html]: add new line

File: vinod/html/ASSIN3/assingment3.q2.html [hunk 17]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline vinod/html/ASSIN3/assingment3.q2.html:1]: remove all extra lines
need proper code indentation
use 4 space in code

File: vinod/html/ASSIN3/link.html [hunk 18]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline vinod/html/ASSIN3/link.html:1]: remove all extra lines
need proper code indentation
use 4 space in code

File: vinod/html/ASSIN3/link.html [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+            akash
+        </li>
+        <li>
+            sai
+        </li>
+    </ul>
+        <ol type="i">
+            <li>balaji</li>
+            <li>omkar</li>  
+            <li>sanket</li>
```
Comment (avinash0070) [inline vinod/html/ASSIN3/link.html:33]: where ol is closed

File: vinod/html/ASSIN3/link.html [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+     </ul>    
+</body>
+</html>
```
Comment (avinash0070) [inline vinod/html/ASSIN3/link.html:36]: add new line

File: vinod/html/ASSIN3/link1.html [hunk 21]
```diff
@@ -0,0 +1,27 @@
...
+    </div>
+
+    
+</body>
+</html>
```
Comment (avinash0070) [inline vinod/html/ASSIN3/link1.html:27]: add extra line

File: vinod/html/ASSIN3/link1.html [hunk 22]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline vinod/html/ASSIN3/link1.html:1]: remove all extra lines
need proper code indentation
use 4 space in code

File: vinod/html/ASSIN4/assingment4.q1.html [hunk 23]
```diff
@@ -0,0 +1,57 @@
...
+  }
+    th, td {
+        border-spacing:15px ;
+    }
+    table {
+  background-color: gray;
+  {
+   caption-background-color:gray;
+   {
+    <caption style="color:purple;">
```
Comment (avinash0070) [inline vinod/html/ASSIN4/assingment4.q1.html:25]: check this line <caption style="color:purple;"> is that work in under Style tag

File: vinod/html/ASSIN4/assingment4.q1.html [hunk 24]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline vinod/html/ASSIN4/assingment4.q1.html:1]: remove all extra lines
need proper code indentation
use 4 space in code

File: vinod/html/ASSIN4/assingment4.q1.html [hunk 25]
```diff
@@ -0,0 +1,57 @@
...
+        </tr>
+            <th>Female</th>
+            <td>1.7</td>
+            <td>0.002</td>
+            <td>43%</td>
+        </tr>
+    </table>
+    
+</body>
+</html>
```
Comment (avinash0070) [inline vinod/html/ASSIN4/assingment4.q1.html:57]: Add new line

File: vinod/html/ASSIN4/assingment4.q2.html [hunk 26]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline vinod/html/ASSIN4/assingment4.q2.html:1]: remove all extra lines
need proper code indentation
use 4 space in code

File: vinod/html/ASSIN4/assingment4.q2.html [hunk 27]
```diff
@@ -0,0 +1,88 @@
...
+        </tr>
+        <tr>
+            <td>4:30</td>
+            <td></td>
+            <td></td>
+            <td></td>
+        </tr>
+     </table>
+  </body>
+</html>
```
Comment (avinash0070) [inline vinod/html/ASSIN4/assingment4.q2.html:88]: Add new line

File: vinod/html/ASSIN4/assingment4.q3.html [hunk 28]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline vinod/html/ASSIN4/assingment4.q3.html:1]: remove all extra lines
need proper code indentation
use 4 space in code

File: vinod/html/ASSIN4/assingment4.q3.html [hunk 29]
```diff
@@ -0,0 +1,85 @@
...
+       <td class="green">12:00 p.m</td>
+       <td> XML Formating Objects</td>
+      </tr>
+      
+        
+		
+		
+    </table>
+</body>
+</html>
```
Comment (avinash0070) [inline vinod/html/ASSIN4/assingment4.q3.html:85]: add new line at the end of the code

File: vinod/html/assignment1/assingment1.Q1.html [hunk 30]
```diff
@@ -0,0 +1,19 @@
...
+</head>
+<body>
+    <h1>HTML</h1> 
+
+    <h3>INTRODUCTION</h3>
+    HTML is standaed markup language for web page.
+    HTML is standaed markup language for web page.
+    HTML is standaed markup language for web page.
+</body>
+</html>>
```
Comment (avinash0070) [inline vinod/html/assignment1/assingment1.Q1.html:19]: add new line

File: vinod/html/assignment1/assingment1.Q1.html [hunk 31]
```diff
@@ -0,0 +1,19 @@
...
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+    <script src='main.js'></script>
+</head>
+<body>
+    <h1>HTML</h1> 
+
```
Comment (avinash0070) [inline vinod/html/assignment1/assingment1.Q1.html:13]: remove extra line

File: vinod/html/assignment1/assingment1.q2.html [hunk 32]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline vinod/html/assignment1/assingment1.q2.html:1]: remove all extra lines
need proper code indentation
use 4 space in code

File: vinod/html/assignment1/assingment1.q2.html [hunk 33]
```diff
@@ -0,0 +1,20 @@
...
+ <body>
+    <h1>HTML</h1>
+   <h3>INTRODUCTION</h3>
+   <p>HTML is standaed markup language for web page.</p>
+   <h1>HTML Editor</h1>
+   <h3>Notpad or TextEdit</h3>
+    <p>web page can be created and modified by using professional HTML editors.<br>
+    However,for learing HTML we recommend a simple text editor like Notpad(pc) or TextEdit(Mac).</p>
+  </body>
+</html>
```
Comment (avinash0070) [inline vinod/html/assignment1/assingment1.q2.html:20]: Add new line

File: vinod/html/assignment1/assingment1.q3.html [hunk 34]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline vinod/html/assignment1/assingment1.q3.html:1]: remove all extra lines
need proper code indentation
use 4 space in code

File: vinod/html/assignment1/assingment1.q3.html [hunk 35]
```diff
@@ -0,0 +1,20 @@
...
+    <h1>HTML</h1>
+    <h3>INTRODUCTION</h3>
+    <p>HTML is standaed markup language for web page.</p>
+    <hr>
+    <h1>HTML Editor</h1>
+    <h3>Notpad or TextEdit</h3>
+    <p>web page can be created and modified by using professional HTML editors.<br>
+        However,for learing HTML we recommend a simple text editor like Notpad(pc) or TextEdit(Mac).</p>
+ </body>
+</html>
```
Comment (avinash0070) [inline vinod/html/assignment1/assingment1.q3.html]: Add new line

File: vinod/html/assignment1/assinngmen Q4.html [hunk 36]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline vinod/html/assignment1/assinngmen Q4.html:1]: remove all extra lines
need proper code indentation
use 4 space in code

File: vinod/html/assignment1/assinngmen Q4.html [hunk 37]
```diff
@@ -0,0 +1,32 @@
...
+    
+    <h1>she is householder. </h1><hr>
+    
+    <h1>my favourite subject is Web development.</h1><hr>
+    
+    <h1>my hobbies are CRICKET.</h1><hr>
+    
+    
+</body>
+</html>
```
Comment (avinash0070) [inline vinod/html/assignment1/assinngmen Q4.html]: Add new line

File: vinod/html/assignment1/assinngmen1. Q4.html [hunk 38]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline vinod/html/assignment1/assinngmen1. Q4.html:1]: remove all extra lines
need proper code indentation
use 4 space in code

File: vinod/html/assignment1/assinngmen1. Q4.html [hunk 39]
```diff
@@ -0,0 +1,32 @@
...
+    
+    <h1>she is householder. </h1><hr>
+    
+    <h1>my favourite subject is Web development.</h1><hr>
+    
+    <h1>my hobbies are CRICKET.</h1><hr>
+    
+    
+</body>
+</html>
```
Comment (avinash0070) [inline vinod/html/assignment1/assinngmen1. Q4.html:32]: Add new line

**Review by avinash0070 — CHANGES_REQUESTED** [review]

**Comment by avinash0070** [comment]
resolve all the comment you done.....otherwise i'm not tracking you with your changes

**Review by avinash0070 — APPROVED** [review]

**Pull request merged**